README.md

```diff
@@ -101,6 +101,16 @@ To execute the tests for this library, run:
 make check
 ```
 
+## Testing
```

Comment:
Duplicate `Testing` title? (Clang tidy is a linter, so shall this title be `Linting` instead?)

Reply:
Changed in https://github.com/scipr-lab/libff/pull/90